Framework/DataHandling/test/SavePARTest.h

```diff
@@ -168,13 +168,6 @@ class SavePARTest : public CxxTest::TestSuite {
     loader.setPropertyValue("Workspace", input);
     loader.execute();
 
-    // mask the detector
-    Geometry::ParameterMap *m_Pmap = &(inputWS->instrumentParameters());
-    boost::shared_ptr<const Instrument> instru = inputWS->getInstrument();
-    Geometry::IDetector_const_sptr toMask = instru->getDetector(THEMASKED);
-    TS_ASSERT(toMask);
-    m_Pmap->addBool(toMask.get(), "masked", true);
-
```

Comment:
Was there a reason to remove the masking here? I guess it might make no difference, the test is clearly a copy paste, but it does at least check nothing bad happens if a masked detector is present.